providers/attio/README.md

```diff
@@ -0,0 +1,16 @@
+# Atio connector
+
+
+## Supported Objects 
+Below is an exhaustive list of the supported Objects in the Attio deep connector with their endpoint resources(ObjectName).
+
+Attio API version : v2
+
+| Object | Resource | Method
```

Comment:
Write seems to be missing as of now?

Reply:
I think objectNames.go is an exact representation of this README. I would remove README, maybe would add comments to that file instead. There is always an incentive to keep go code up to date unlike `.md`, `.txt` type of files.